contrib/emptty/patches/dinit-service.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/res/dinit-service b/res/dinit-service
```

Comment:
write this out manually in files/ and install it from there

Reply:
Done